examples/step-63/step-63.cc

```diff
@@ -814,9 +814,9 @@ namespace Step63
     for (unsigned int level = 0; level < triangulation.n_global_levels();
          ++level)
       {
-        const IndexSet locally_owned_level_dof_indices =
-          DoFTools::extract_locally_relevant_level_dofs(dof_handler, level);
```

Comment:
This stumped me briefly: The variable is called `locally_OWNED_...`, but it is initialized by `extract_locally_RELEVANT_...()`. Clearly a mistake in naming :-)